src/lib/Components/Consignments/Screens/Edition.tsx

```diff
@@ -53,48 +50,58 @@ export default class Edition extends React.Component<Props, ConsignmentSetup> {
 
   render() {
     return (
-      <ConsignmentBG>
-        <DoneButton onPress={this.doneTapped}>
-          <Form>
-            <Row>
-              <Label>Is this an edition?</Label>
-              <Toggle selected={!!this.state.editionInfo} left="YES" right="NO" onPress={this.updateEdition} />
-            </Row>
-
-            {this.state.editionInfo ? (
-              <Row>
-                <Text
-                  text={{
-                    placeholder: "Edition Size",
-                    keyboardType: "phone-pad",
-                    onChangeText: this.updateEditionSize,
-                    value: this.state.editionInfo && this.state.editionInfo.size,
-                  }}
-                  style={{ margin: 10 }}
-                />
-                <Text
-                  text={{
-                    placeholder: "Edition Number",
-                    onChangeText: this.updateEditionNumber,
-                    value: this.state.editionInfo && this.state.editionInfo.number,
-                  }}
-                  style={{ margin: 10 }}
-                />
-              </Row>
-            ) : null}
-
-            <Row>
-              <Label>Is this work signed?</Label>
-              <Toggle selected={this.state.signed} left="YES" right="NO" onPress={this.updateSigned} />
-            </Row>
-
-            <Row>
-              <Label>Do you have a certificate of authenticity?</Label>
-              <Toggle selected={this.state.certificateOfAuth} left="YES" right="NO" onPress={this.updateCert} />
-            </Row>
-          </Form>
-        </DoneButton>
-      </ConsignmentBG>
+      <Theme>
+        <BottomAlignedButton onPress={this.doneTapped} buttonText="Done">
+          <Flex style={{ flex: 1 }} p={1}>
+            <Form>
+              <Flex justifyContent="space-between" alignItems="center" flexDirection="row" flexWrap="nowrap">
+                <Flex style={{ flex: 1 }}>
+                  <Serif size="5">Is this an edition?</Serif>
+                </Flex>
+                <Toggle selected={!!this.state.editionInfo} left="Yes" right="No" onPress={this.updateEdition} />
+              </Flex>
+              <Spacer mb={2} />
+              {this.state.editionInfo ? (
+                <>
+                  <Row>
+                    <Text
+                      text={{
+                        placeholder: "Edition Size",
```

Comment:
With sentence casing, would this be more appropriate?

```suggestion
                        placeholder: "Edition size",
```